playbooks/06-deploy-architecture.yml

```
@@ -197,6 +197,14 @@
         - update_containers
         - edpm_bootstrap
 
+    - name: Update containers in deployed OSP operators using set_openstack_containers role
+      when: cifmw_set_openstack_containers | default(false)
```

```suggestion
      when: cifmw_set_openstack_containers | default(false) | bool
```

Using bool filters to "cast" public API variables to ensure type safety is best practice.

---

done